packages/admin-ui-docs/docs/experimental/filter-group.mdx

```diff
@@ -5,12 +5,16 @@ path: /filter-group/
 
 # FilterGroup
 
+Filters are supposed to be used in pages or sections that display some kind of data that can be filtered. The `FilterGroup` 
```

Comment:
Should the first phrase be kept here in the FilteGroup context? Or moved up to the Filters' context?

Reply:
I'm not sure what you're suggesting. The Filter page already has this, so I should remove it here?

Reply:
What do you mean by "Filters context"?